hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -126,7 +143,7 @@ function ozonecmd_case
     ;;
     freon)
       OZONE_CLASSNAME=org.apache.hadoop.ozone.freon.Freon
-      OZONE_FREON_OPTS="${OZONE_FREON_OPTS}"
+      OZONE_FREON_OPTS="${OZONE_FREON_OPTS} ${OZONE_MODULE_ACCESS_ARGS}"
```

Comment:
@devmadhuu , is freon or Recon?

Reply:
> @devmadhuu , I think we shall add the OZONE_MODULE_ACCESS_ARGS for Recon too.

Thanks @ChenSammi , I have added as per your suggestion. Pls re-review.